source/_variables/settings.py

```diff
@@ -21,7 +21,7 @@
 # The full version, including alpha/beta/rc tags
 # Important: use a valid branch (4.0) or, preferably, tag name (v4.0.0)
 
-release = '4.11.1'
-api_tag = '4.11.1'
+release = '4.11.0'
+api_tag = 'v4.11.0'

```

Comment:
```suggestion
release = '4.11.1'
api_tag = '4.11.1'
```

Reply:
When in development, there's no tag such as `v4.11.1` so we need to use the branch name such as `4.11.1` as the reference. When we release, we switch the `v4.11.1` reference because then a tag is available.